common/web/base-web/src/dev/webpack/prod.ts

```diff
@@ -90,6 +91,14 @@ const plugins = [
     statsFilename: 'iconstats.json',
   }),
 
+  new CopyWebpackPlugin([
```

Comment:
Let's put this in `base.ts` - it's worth being able to confirm the assets with the dev server.